### PR TITLE
fix runaway check function

### PR DESCRIPTION
**Pull request by joekiller**
bind(this) always returns a new function reference which is why it wouldn't unbind

https://stackoverflow.com/a/22870717/2240967

### CODE REVIEW

File: src/classes/Listings/Listings.ts [hunk 1]
```diff
@@ -57,26 +59,19 @@ export default class Listings {
...
 
     disableAutorelistOption(): void {
-        if (!this.autoRelistEnabled) {
-            // If the process already not running, just don't execute this.
-            return;
-        }
-        this.bot.listingManager.removeListener('heartbeat', this.checkAccountInfo.bind(this));
-        this.bot.listingManager.off('heartbeat', this.checkAccountInfo.bind(this));
-        this.isForceDisabled = true;
-        this.disableAutoRelist();
```
Comment (idinium96) [inline src/classes/Listings/Listings.ts:79]: Thanks, but I wonder why you remove this and replace it with the same thing inside this function.

Comment (idinium96) [reply src/classes/Listings/Listings.ts:79]: I mean, `this.disableAutoRelist();`
https://github.com/TF2Autobot/tf2autobot/blob/58f6ee85c5ffb0d1c7a0d5b71115ba2088666b0a/src/classes/Listings/Listings.ts#L150-L155

Comment (joekiller) [reply src/classes/Listings/Listings.ts:79]: I think I was just leaving it how it was prior to the attempted fixes. I cleaned it up and put it through disableAutoRelist now

**Comment by idinium96** [comment]
tested and it's working
image

**Pull request merged**